docs/sql/statements/create-table.rst

```diff
@@ -825,7 +825,9 @@ shard before giving up and leaving the shard unallocated.
 ------------------------------------------
 
 Assign the table to a node whose ``{attribute}`` has at least one of the
-comma-separated values.
+comma-separated values. This setting overrides the corresponding
```

Comment:
minor: for consistency, I'd suggest to use either `corresponding` or `related` ;) 
```suggestion
comma-separated values. This setting overrides the related
```